Assets/Scripts/Entities/Order.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System.Collections.Generic;
+
+public class Order
+{
+    private int orderId { get; set; }
+    private int customerId { get; set; }
+    private OrderState orderState { get; set; }
+    private List<OrderItem> items { get; }
+
+    private bool isSuccess = false;
+
+    public Order(int orderId)
+    {
+        this.orderId = orderId;
```

Comment:
Instead of giving an orderId, the class could have a static object orderIdIncrement?